server/app/services/export/JsonPrettifier.java

```diff
@@ -6,16 +6,38 @@
 
 /** Provides methods related to "prettifying" (formatting) JSON strings. */
 public final class JsonPrettifier {
-  public static String asPrettyJsonString(String original) {
+
+  /**
+   * Pretty-print the JSON document
+   *
+   * @param jsonObject the JSON document as an {@link Object}
+   * @return the pretty-printed document
+   */
+  public static String asPrettyJsonString(Object jsonObject) {
     ObjectMapper objectMapper = new ObjectMapper();
     objectMapper.enable(SerializationFeature.INDENT_OUTPUT);
     objectMapper.enable(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS);
 
     try {
-      Object jsonObject = objectMapper.readValue(original, Object.class);
       return objectMapper.writerWithDefaultPrettyPrinter().writeValueAsString(jsonObject);
     } catch (JsonProcessingException e) {
       return "Error parsing json";
     }
   }
+
+  /**
+   * Pretty-print the JSON document
+   *
+   * @param jsonString the JSON document as a {@link String}
+   * @return the pretty-printed document
+   */
+  public static String asPrettyJsonString(String jsonString) {
+    ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
**question (blocking):** why not have a single static `ObjectMapper` so that the options on it are always consistent wherever it's used?

Reply:
Done!